mrmustard/utils/xptensor.py

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# pylint: disable=too-many-return-statements,too-many-branches,missing-function-docstring
```

Comment:
Would be better to actually add missing docstrings here rather than just disabling the Pylint message. The other two might require more refactoring though.